doc/debugging-dmtcp.txt

```diff
@@ -23,6 +23,15 @@ the cause, try:
 
 ===
 
+JASSERT doesn't generate a coredump by default (because it calls _exit()).
+Exporting the environment variable DMTCP_ABORT_ON_FAILED_ASSERT will cause
+it to call abort(). This can be useful when debugging distributed processes.
+Example:
+  DMTCP_ABORT_ON_FAILED_ASSERT=1 bin/dmtcp_launch a.out
+  gdb -c core
```

Comment:
1. Let's change "gdb -c core" to "gdb a.out core".  It's safer, especially if this was a restarted process.
2. Then, let's add a further statement -- something like:

    When used with dmtcp_restart, it is important to embed DMTCP_ABORT_ON_FAILED_ASSERT
    in the checkpoint image by first using it with dmtcp_launch, as above.  After that, a
    restarted process will also generate a core dump.```